packages/db-base/package.json

```diff
@@ -33,7 +33,19 @@
   "publishConfig": {
     "access": "public"
   },
+  "exports": {
+    ".": "./build/index.js",
+    "./package.json": "./package.json",
+    "./tools": "./build/lib/inMemFileDB.js"
```

Comment:
```suggestion
    "./inMemFileDB": "./build/lib/inMemFileDB.js"
```

Reply:
LOL forgot this one, only adjusted in types 💩

Reply:
Good thing that VSCode complains about the input trying to use this :)